scripts/consensus.py

```diff
@@ -33,14 +38,171 @@ def concatenate_data(profile_list: list) -> pd.DataFrame:
         concat_df.columns.str.startswith("Metadata")
     ]
     concat_metadata_df = concat_df.loc[:, concat_metadata_cols]
+
     concat_df = concat_df.drop(concat_metadata_cols, axis="columns")
-    concat_df = pd.concat([concat_metadata_df, concat_df])
+    concat_df = pd.concat([concat_metadata_df, concat_df], axis="columns")
 
     # dropping columns with na values
     na_cols = get_na_columns(concat_df, cutoff=0)
     concat_df = concat_df.drop(na_cols, axis="columns")
 
-    return concat_df
+    # dropping Costes Features
+    costes_cols_to_drop = [x for x in concat_df.columns if "costes" in x.lower()]
+    concat_df = concat_df.drop(costes_cols_to_drop, axis="columns")
+
+    x_groupby_cols = ["Metadata_gene_name", "Metadata_pert_name", "Metadata_cell_line"]
```

Comment:
This needs to be a customizable argument. Not all of these columns will always be present, and sometimes we want to aggregate different columns.

Reply:
The whole `consensus.py` contains lots of hard coded parameters. Will submit this as an issue

Reply:
Issue created: #12

Reply:
good to open the issue, but I don't think we should merge hardcoded variables into main. In other words, it's best to close #12 in this PR.